test/documentation/seqan3-doxygen.cmake

```diff
@@ -41,7 +41,7 @@ ExternalProject_Add (
     DOWNLOAD_NAME "html-book.tar.xz"
     DOWNLOAD_NO_EXTRACT YES
     BINARY_DIR "${PROJECT_BINARY_DIR}"
-    CONFIGURE_COMMAND tar -xf "html-book.tar.xz" "cppreference-doxygen-web.tag.xml"
+    CONFIGURE_COMMAND /bin/sh -c "xzcat html-book.tar.xz | tar -xf - cppreference-doxygen-web.tag.xml"
```

Comment:
Why this change?

I was a bit confused until I looked at the change log. Usually need to install xz and run make clean to build it.

Reply:
> Why this change?

OpenBSD tar doesnt have builtin support for xz decompression.

> I was a bit confused until I looked at the change log. Usually need to install xz and run make clean to build it.

What do you mean?

Reply:
@smehringer and I first got the error that `xzcat` is not available. So we installed `xz`. Then it complained about some file. `make clean` fixed it.

I was just wondering why it was changed when it seemingly causes (one time) troubles when building locally.